lib/styles/settings/typography/_tokens.scss

```diff
@@ -0,0 +1,46 @@
+$typography-font-family: var(--typography-font-family);
+
+$typography-font-size-base: var(--typography-font-size-base);
+$typography-font-size-s: var(--typography-font-size-s);
+$typography-font-size-xs: var(--typography-font-size-xs);
+$typography-font-size-2xs: var(--typography-font-size-2xs);
+
+$typography-font-weight-bold: var(--typography-font-weight-bold);
+
+$typography-line-height-2xs: var(--typography-line-height-2xs);
+$typography-line-height-3xs: var(--typography-line-height-3xs);
+$typography-line-height-4xs: var(--typography-line-height-4xs);
+
+$typography-letter-spacing-s: var(--typography-letter-spacing-s);
+
+body {
+  font-family: $typography-font-family;
+  font-size: $typography-font-size-base;
+}
+
+@mixin labelLDefaultBold() {
+  font-family: $typography-font-family;
```

Comment:
I'm not convinced that we need `font-family` in tokens. Why do we need it? Is it in JSON file provided by UX?

Reply:
I think we can add condition for default font-family. If it will be our default `Lato` - we will not add it, otherwise we will add. So I'm removing it, it's redundant for me.

Reply:
let's remember to communicate that to UX